docker-compose-arm64.yml

```
@@ -30,7 +34,12 @@ services:
         # domain:
         # https_only: false
         # statistics_enabled: false
-        hmac_key: "CHANGE_ME!!"
+        hmac_key: "DEVELOPMENT_KEY__DO_NOT_USE"
+        invidious_companion_key: "DEVELOPMENT__KEY"
```

```suggestion
        hmac_key: "DEVELOPMENT_KEY__DO_NOT_USE"
        invidious_companion_key: "DEVELOPMENT__KEY"
```

Considering the amount of people that just blindly runs the compose file from the repo I'll rather not make their Invidious instances less secure than we have to. Since at least for `CHANGE_ME!!` Invidious will exit and force a different value.

Can we just have the Github actions runner `sed` that particular line to something else?